include/seqan3/range/container/bitcompressed_vector.hpp

```diff
@@ -65,7 +66,7 @@ class bitcompressed_vector
 {
 private:
     //!\brief The number of bits needed to represent a single letter of the alphabet_type.
-    static constexpr size_t bits_per_letter = std::ceil(std::log2(alphabet_size<alphabet_type>));
+    static constexpr size_t bits_per_letter = detail::next_power_of_two(alphabet_size<alphabet_type>);
```

Comment:
```suggestion
    static constexpr size_t bits_per_letter = detail::next_power_of_two(1ULL << detail::most_significant_bit_set(alphabet_size<alphabet_type>));
```

Should be correct. Can you check the indentation?